index.html

```diff
@@ -59,22 +59,36 @@ <h2>A project for first-time contributions</h2>
 
        </div>
        <!-- ________ Contributor card END ________  -->
-       
-           
+
+
         <!-- ^^^^ Paste YOUR CARD directly ^^ ABOVE ^^ this line ^^^^ -->
         <!-- ^^^^ make sure there is one line of space between your card start and the card end above it ^^^^ -->
         <!-- ^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^ -->
         <!-- ^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^ -->
-        
-         
-         
+
+        <div class="card">
+
+            <p class="name">Paula Cardoso</p>
+            <p class="contact"><i class="fa fa-twitter"></i> <a href="YOUR twitter account link">@yourHandle</a></p>
+            <p class="about">I'm Paula, a portuguese journalist and soon, I hope, a web developer...</p>
+            <div class="resources">
+                <p>3 Useful Dev Resources</p>
+                <ul>
+                    <li><a href="#" target="https://developers.google.com/web/" title="Build next generation web experiences. ">Google Developers</a></li>
```

Comment:
Please remove the website link from `target` to `href`
You should remove the `#` in `href="#"` and replace it with the link to the website `https://.....etc...`
Your `target` should be returned to the original state `target="_blank"`
You should do this to the 3 links in the resources.
Example. Your first resource should look like this:
`href="https://developers.google.com/web/" target="_blank"`

Reply:
Once you make the changes you only need to repeat steps 6 & 7 of the tutorial, commit and then push your changes. And that's all. I will be notified and if everything is good I will merge your card 🙂